distribution/tools/server-cli/src/test/java/org/elasticsearch/server/cli/JvmOptionsParserTests.java

```diff
@@ -353,32 +353,35 @@ public void accept(final int lineNumber, final String line) {
 
     public void testNodeProcessorsActiveCount() {
         {
-            final List<String> jvmOptions = SystemJvmOptions.systemJvmOptions(Settings.EMPTY, TEST_SYSPROPS);
+            final List<String> jvmOptions = SystemJvmOptions.systemJvmOptions(Settings.EMPTY, TEST_SYSPROPS, Path.of(""));
```

Comment:
I'd add some tests to cover for a non-empty path

Reply:
Heh, I guess at a broader level, I should be adding _at least one test_ for the logic I just added. 😅